src/lib/feature/user/presentation/user_detail_screen.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:yamabico/component/audio_post.dart';
+import 'package:yamabico/component/my_app_bar.dart';
+import 'package:yamabico/component/my_music_player.dart';
+import 'package:yamabico/feature/posts/presentation/index_screen.dart';
+
+class UserDetailScreen extends StatelessWidget {
+  final User user;
+  final List<AudioData> userPosts;
+
+  const UserDetailScreen({
+    Key? key,
+    required this.user,
+    required this.userPosts,
+  }) : super(key: key);
```

Comment:
[nits]

Dart2.17から以下のようにシンプルな書き方ができるようになったみたいです！

```suggestion
    const UserDetailScreen({
      super.key,
      required this.user,
      required this.userPosts,
  });

```

Reply:
ありがとうございます！対応しました